snuba/cli/query_replayer.py

```diff
@@ -56,6 +62,15 @@ def get_credentials(user: Optional[str], password: Optional[str]) -> Tuple[str,
     return (user or "default", password or "")
 
 
+# Kind of hacky but the saved schemas will have one
+# of these clusters defined
+CLUSTERS = {
+    "snuba-st",
+    "snuba-generic-metrics",
+    "snuba-shared",
+}
```

Comment:
Could you elaborate why this is needed at all. Why can't the CLI arguments explicitly provide this information?

Reply:
I can add it as a CLI arg - was just trying to limit how many args there were, and trying to make it easier to not think about what the previous cluster name was, just to know the current one (e.g. `snuba-test`) but i can update

Reply:
updated